common/display/displayplanemanager.cpp

```diff
@@ -235,12 +236,22 @@ bool DisplayPlaneManager::ValidateLayers(
                 if (!FallbacktoGPU(last_plane.GetDisplayPlane(),
                                    last_plane.GetOffScreenTarget()->GetLayer(),
                                    commit_planes)) {
+                  force_all_layers = false;
                   validate_final_layers = true;
                 }
               }
             }
-
-            break;
+            if (force_all_layers) {
```

Comment:
Probably you can revert all changes related to displayplanemanager from the bad commit ?